README.md

```diff
@@ -228,6 +228,27 @@ It simply chooses which version of the data to seed (the Snowflake copy should c
 **Args:**
 * `seed_name` (required): Name of the seed that has separate snowflake seed data.
 
+----
+### staging_models_automation ([source](macros/staging_models_automation.sql))
+This macro is intended to be used as a `run-operation` when generating Fivetran dbt source package staging models/macros. This macro will receive user input to create all necessary ([bash commands](columns_setup.sh)) appended with `&&` so they may all be ran at once. The output of this macro within the CLI will then be copied and pasted as a command to generate the staging models/macros.
+
+**Usage:**
+```bash
+dbt run-operation staging_models_automation --args '{package: asana, target_schema: asana_target, target_database: database-target-name, tables: ["user","tag"]}'
```

Comment:
got a little confused by the use of `target_*`. in this case, the target_schema is the source schema we're reading from right? and not the dbt target.schema we're writing these models to? if so, maybe we'd wanna use a different naming convention

Reply:
or we could add some more detailed description to the args section below

Reply:
Nope you are completely right! I meant to use source instead of target for the schema and database, thanks I will update it to reflect source!

Reply:
Updated the macro and readme to reflect source opposed to target.